algebra-testing/src/main/java/com/hubspot/assertj/algebra/error/ResultShouldBeErrWithValue.java

```diff
@@ -10,11 +10,13 @@ private ResultShouldBeErrWithValue(String format, Object... arguments) {
     super(format, arguments);
   }
 
-  public static <T, E> ErrorMessageFactory shouldBeOkWithValue(Result<T, E> actual, Object value) {
+  public static <T, E> ErrorMessageFactory shouldBeErrWithValue(Result<T, E> actual, Object error) {
```

Comment:
as said above, we're comparing that a result is equal to the expected Error result